libbeat/cmd/instance/beat.go

```diff
@@ -215,7 +209,7 @@ func NewBeat(name, indexPrefix, v string) (*Beat, error) {
 			Hostname:    hostname,
 			UUID:        id,
 		},
-		Fields: fields,
+		//Fields: fields,
```

Comment:
We should remove this.. following the change in the struct.

Reply:
Sure, once we decided to remove in the struct, I'll also remove it here. ATM both is commented out as discussions are going on.